recipes/xtb-python/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "xtb-python" %}
+{% set version = "20.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/grimme-lab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: fc0cfad603324c4a41c575c7c3bf671bf68f03b1b9f0e40e8dc501a56b1b9b1e
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - meson >=0.51
+    - ninja
+    - python
+    - cffi
```

Comment:
I suspect these aren't doing anything (it almost never makes sense to have the same thing in `build` and `host`):
```suggestion
```

Reply:
Congratulations, I think you need `cffi` in `build` but not `host` 🤞

Reply:
Seems like Python and CFFI are needed in the progress of building the C extension module.

Without the Python dependency the extension module will be build against the systems Python version rather than the cf Python version, which lead to a missmatch in the package later.

Reply:
I certainly need CFFI in host as well because `__init__.py` will always `import cffi` for now.

Reply:
Removing cffi from the host env worked.